### PR TITLE
Balancing/is balanced

### PR DESCRIPTION
**Pull request by casparschucan**
provides is_balanced util

### CODE REVIEW

File: src/OrthoTree/helpers/MortonHelper.hpp [hunk 1]
```diff
@@ -353,4 +353,31 @@ namespace ippl {
...
+
+        auto neighbors = get_neighbors(a, get_depth(a));
+
+        for (auto& neighbor : neighbors) {
+          if (neighbor == b) {
+            return true;
+          }
+          
+          if (is_descendant(neighbor, b)) {
+            return true;
```
Comment (mazzeaar) [inline src/OrthoTree/helpers/MortonHelper.hpp:376]: is this correct?

**Review by mazzeaar — APPROVED** [review]
see comment, then merge

File: src/OrthoTree/helpers/MortonHelper.hpp (hunk 1 above)
Comment (casparschucan) [reply src/OrthoTree/helpers/MortonHelper.hpp:376]: Argument is if they are neighbors the finer octant will have some fictitious neighbor octant that is inside the coarser octant.

Comment (mazzeaar) [reply src/OrthoTree/helpers/MortonHelper.hpp:376]: makes sense:)

**Pull request merged**